Terminal.Gui/Drawing/Scheme.cs

```diff
@@ -306,7 +306,7 @@ private Attribute GetAttributeForRoleCore (VisualRole role, HashSet<VisualRole>
                                VisualRole.Focus =>
                                    GetAttributeForRoleCore (VisualRole.Normal, stack) with
                                    {
-                                       Foreground = GetAttributeForRoleCore (VisualRole.Normal, stack).Background,
+                                       Foreground = SwapTransparentForBlack(GetAttributeForRoleCore (VisualRole.Normal, stack).Background),
```

Comment:
Otherwise a text in a transparent foreground with a opaque background is unreadable, correct?